### ISSUE TITLE
Ectra.Text emoji and unicode support

### ISSUE THREAD

**Issue by nicholasvperry** [threlte/threlte#830]
Emoji and Unicode is not working in the threlte/extra text component. Below I have a component where I am trying to pass simple emojis in but it will not render. I was able to load an emoji font and make it work, however I cannot then have text and emoji when using this font.

```
<script>
      import * as Extra from '@threlte/extras';
      import { InteractiveObject, OrbitControls, T, useTexture, useThrelte } from '@threlte/core';
</script>


<T.Group>
		<T.Mesh position.x={$tLongitude} position.y={$tHeight} position.z={$tLatitude} let:ref>
			<InteractiveObject
				object={ref}
				interactive
				on:click={() => showModal()}
				on:pointerenter={onPointerEnter}
				on:pointerleave={onPointerLeave}
			/>
			<T.BoxGeometry args={[boxSize.length, 0.2, boxSize.width]}>
				<
				<Extra.Text
				text="⬅️"
				fontSize={0.23}
				color="black"
				/>
				
			</T.BoxGeometry>
			<T.MeshStandardMaterial map={texture} metalness="2" />
		</T.Mesh>
	</T.Group>
```

**Comment by grischaerbe** [comment]
Hi @nicholasvperry,

this is a limitation of the library used to display text. Behind the scenes [`troika-three-text`](https://github.com/protectwise/troika) converts the characters of the font in use to [SDFs](https://en.wikipedia.org/wiki/Signed_distance_function). Feel free to report in the upstream repository 👍

**Comment by nicholasvperry** [comment]
Thanks @grischaerbe. I did a report upstream!

**Comment by nicholasvperry** [comment]
It appears as though Threlte is not using the most current version of troika. Updating to the newest version would fix this problem.

**Comment by grischaerbe** [comment]
Fixed in #831  🎉
